tuxemon/item/item.py

```diff
@@ -56,6 +58,8 @@ def __init__(self, save_data: Optional[Mapping[str, Any]] = None) -> None:
         self.quantity = 1
         self.images: Sequence[str] = []
         self.type = ItemType.consumable
+        self.animation = Optional[str]
```

Comment:
Wrong way of adding typehint.

Should be `self.animation: Optional[str] = None`.
Same issue for the already existing statement above.

Reply:
Btw why is it of type `str` instead of `Animation` or `Task`?

Looks like it's not really the animation but only the location for the sprites, so maybe it should be renamed `animation_sprites_location`, `frames_path` or something like this.

Reply:
copy paste issue, regarding the name, I agree, but there is another PR touching that spot both in Technique and Condition

Reply:
Yes don't worry I knew it was copy paste issue.